pkg/sql/instrumentation_test.go

```diff
@@ -251,10 +252,11 @@ func TestSampledStatsCollectionOnNewFingerprint(t *testing.T) {
 
 		require.Equal(t, len(queries), len(collectedTxnStats))
 
-		// We should have collected stats for each of the queries.
-		for i := range collectedTxnStats {
+		// We should have collected stats for each of the first 4 queries.
```

Comment:
nit: maybe change this to "each of the queries, except the last" or something, since more queries could be added in the future